tests/utils.go

```diff
@@ -476,9 +478,13 @@ func WaitForVirtualMachineStatus(client kubecli.KubevirtClient, namespace, name
 
 		for _, status := range statuses {
 			if vm.Status.PrintableStatus == status {
+				fmt.Fprintf(ginkgo.GinkgoWriter, " got %s\n", status)
+
 				return true, nil
 			}
 		}
+		fmt.Fprint(ginkgo.GinkgoWriter, ".")
```

Comment:
This seems odd, just printing a dot.

Reply:
looks nice, and helps to estimate how long it was waiting, but it is totally not needed. I can probably remove it.

Reply:
removed